huaweicloud/services/cce/resource_huaweicloud_cce_node_pool.go

```diff
@@ -181,6 +181,128 @@ func ResourceCCENodePool() *schema.Resource {
 				Sensitive:    true,
 				ExactlyOneOf: []string{"password", "key_pair"},
 			},
+			"storage": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"selectors": {
+							Type:     schema.TypeList,
+							Required: true,
+							ForceNew: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type:     schema.TypeString,
+										Required: true,
+										ForceNew: true,
+									},
+									"type": {
+										Type:     schema.TypeString,
+										Optional: true,
+										ForceNew: true,
+										Default:  "evs",
```

Comment:
remove the default value?

Reply:
this is a required parameter, but only evs supported, so use a default value is good here